### PR TITLE
MM-1787 Fixing links for signup from team invite link.

### PR DESCRIPTION
**Pull request by crspeller**
- Fixed team invite link so it does not try to use the teamURL and instead uses the siteURL.
- Enter now advances signup_user_complete screen.
- Sign in link now redirects properly and fills in your email for you.

### CODE REVIEW

File: web/templates/verify.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
         </div>
     </div>
     <script>
-        window.setup_verify_page('{{ .Props.IsVerified }}');
+window.setupVerifyPage('{{.Props.IsVerified}}', '{{.Props.TeamURL}}', '{{.Props.UserEmail}}');
```
Comment (jwilander) [inline web/templates/verify.html]: Indentation looks off here

**Comment by jwilander** [comment]
Other than my minor comment, +1

File: web/templates/verify.html (hunk 1 above)
Comment (crspeller) [inline web/templates/verify.html]: My vim hates html files. It's always trying to do that to any JS in the html files.

**Comment by hmhealey** [comment]
We can probably remove the nickname field that's being passed through FireAndForgetVerifyEmail into the email template since that users won't actually have a nickname set at this point in the user creation process. Even if it was somehow set, the template actually makes no use of the nickname anyway.

**Comment by crspeller** [comment]
@hmhealey Removed nickname from FireAndForgetVerifyEmail. Ready for re-review.

**Comment by hmhealey** [comment]
Cool, looks good.

**Pull request merged**